internal/server/config.go

```diff
@@ -603,8 +603,14 @@ func (s Server) loadConfig(config Config) error {
 	if err := validate.Validate(config); err != nil {
 		return err
 	}
+	org, err := data.GetOrganization(s.DB(), data.ByName(models.DefaultOrganizationName))
```

Comment:
We got rid of the ability to set the default organization name in config, correct? Just validating.

Reply:
I guess we did.